core/templates/dev/head/services/AssetsBackendApiService.js

```diff
@@ -216,6 +219,9 @@ oppia.factory('AssetsBackendApiService', [
             ASSET_TYPE_AUDIO)) {
             _fetchFile(explorationId, filename, ASSET_TYPE_AUDIO,
               resolve, reject);
+          } else {
```

Comment:
Does it make sense to combine this clause with the previous one? Ditto below.

Reply:
Yes, that would be better.